nav2_loopback_sim/nav2_loopback_sim/loopback_simulator.py

```diff
@@ -115,13 +156,15 @@ def initialPoseCallback(self, msg):
             self.t_odom_to_base_link.transform.translation = Vector3()
             self.t_odom_to_base_link.transform.rotation = Quaternion()
             self.publishTransforms(self.t_map_to_odom, self.t_odom_to_base_link)
+            self.mat_map_to_odom = transformStampedToMatrix(self.t_map_to_odom)
```

Comment:
These are non-static, do not store, get when needed. We don't want to have duplicate information we're trying to keep consistent.